libraries/SITL/SITL.cpp

```diff
@@ -58,16 +58,36 @@ const AP_Param::GroupInfo SIM::var_info[] = {
     AP_GROUPINFO("DRIFT_SPEED",    5, SIM,  drift_speed, 0.05f),
     AP_GROUPINFO("DRIFT_TIME",     6, SIM,  drift_time,  5),
     AP_GROUPINFO("ENGINE_MUL",     8, SIM,  engine_mul,  1),
+    // @Param: RC_FAIL
+    // @DisplayName: Simulated RC signal failure
+    // @Description: Allows you to emulate rc failures in sim
+    // @Values: 0:Disabled,1:No RC pusles,2:Binding pulses
+    // @User: Advanced
+    AP_GROUPINFO("RC_FAIL",       25, SIM,  rc_fail, 0),
```

Comment:
Why are you moving this parameter definition?

As a pattern it would be nice to annotate them in-place as a single PR and move them as a separate PR if there's a good reason.  My thinking there is that it makes it easier to merge documentation-only PRs from random people then.

Reply:
to get the wind params all together instead of being split by RC fail